CRM/Utils/System/Base.php

```diff
@@ -1114,4 +1114,13 @@ public function checkCleanurls() {
     return [];
   }
 
+  /**
+   * Suppress profile form errors
+   *
+   * @return bool
+   */
+  public function suppressProfileFormErrors() {
```

Comment:
wonder if its useful to add a `:bool` type hint onto these functions

Reply:
sure